_projects/newcomer_metrics.md

```diff
@@ -13,6 +13,7 @@ name: "Newcomer metrics and gamification"
 requirements:
   - "At least one patch to each of coala-landing and coala-html should be accepted and merged."
   - "The student should be familiar with web design tools & trends."
+prereq: "Add PRs to coala/community repository https://github.com/coala/community/ "
```

Comment:
@himanshu1603 , you don't have to create another section for `prereq`, basically this issue  means that you have to replace the first requirement of the project e.g: `At least one patch to each of coala-landing and coala-html should be accepted and merged.` will be something like `The student should have patches in coala/community repo`